tests/ts/lsfd/lsfd-functions.bash

```diff
@@ -92,3 +93,30 @@ function lsfd_check_mkfds_factory
 		ts_skip "test_mkfds has no factory for $FACTORY"
 	fi
 }
+
+function lsfd_skip_or_failed_test_if_no_sockdiag
```

Comment:
`ts_lsfd_check_sockdiag`?

Reply:
Should we use `ts_` as a prefix?
This function is used only in test cases for lsfd, so using `lsfd_` looks safe.

Reply:
I prefer the `ts_` prefix slightly. But both are fine. Just the current name is very unwieldy.